analyser_merge_milestone_FR_metropole.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights Frédéric Rodrigo 2014-2020                                 ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from collections import OrderedDict
+from .Analyser_Merge import Analyser_Merge, Source, CSV, Load, Mapping, Select, Generate
+
+class Analyser_Merge_Milestone_FR_metropole(Analyser_Merge):
+    def __init__(self, config, logger = None):
+        Analyser_Merge.__init__(self, config, logger)
+
+        # fix item 
+        self.def_class_missing_official(item = 8130, id = 41, level = 3, tags = ['merge', 'highway'],
+            title = T_('Milestone not integrated'))
+        self.def_class_possible_merge(item = 8131, id = 43, level = 3, tags = ['merge', 'highway'],
+            title = T_('Milestone integration suggestion'))
+        self.def_class_update_official(item = 8132, id = 44, level = 3, tags = ['merge', 'highway'],
+            title = T_('Milestone update'))
```

Comment:
Can you add a bit of documentation
https://github.com/osm-fr/osmose-backend/blob/master/doc/0-Index.md#classes-of-issues

Could be like this
https://github.com/osm-fr/osmose-backend/blob/master/analysers/analyser_merge_heritage_FR_merimee.py

Reply:
ok, 
it has nothing to do with it but i noticed in heritage_FR_merimee,
this code 
Analyser_Merge.__init__(self, config, logger)
is present a line 30 and 31 ??

Reply:
Thank I will fix it.